### ISSUE TITLE
[Tournaments API] Bugged timestamps

### ISSUE THREAD

**Issue by Gr8z** [cr-api/cr-api#219]
Can we revert back to the Unix timestamp 💃 
```
"createTime": "1970-01-18 12:50:12",
"startTime": "1970-01-18 12:50:19",
"endTime": "1970-01-18 12:50:23",
```
My tourneys are suddenly going back to 1970s.

**Comment by Gr8z** [comment]
timestamps are fixed but now they have quotes inside of quotes. Don't know if its a bug or not.

**Comment by selfish** [comment]
Fixed in `v8.4.35` - was a bug.
